ivy/functional/frontends/paddle/fft.py

```diff
@@ -16,6 +16,31 @@ def fft(x, n=None, axis=-1.0, norm="backward", name=None):
     return ivy.astype(ret, x.dtype)
 
 
+@with_supported_dtypes(
+    {
+        "2.5.1 and below": (
+            "int32",
+            "int64",
+            "float32",
+            "float64",
+        )
+    },
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def fftfreq(n, d=1.0, dtype=None, name=None):
+    if d * n == 0:
+        raise ValueError("d or n should not be 0.")
+
+    dtype = ivy.default_dtype()
```

Comment:
I think you need to check if it `None` or not before doing that

Reply:
Hey! Sorry for the late response. I have added the dtype check.